lib/configParser.js

```diff
@@ -58,9 +90,8 @@ var ConfigParser = function() {
  */
 var merge_ = function(into, from) {
   for (var key in from) {
-    if (into[key] instanceof Object &&
-        !(into[key] instanceof Array) &&
-        !(into[key] instanceof Function)) {
+    if ((isArrayLike_(into[key]) && isArrayLike_(from[key])) || 
+        (isDictionary_(into[key]) && isDictionary_(from[key]))) {
```

Comment:
Discussed offline: We think that overriding values in an object make sense (e.g. `--jasmineNodeOpts.showColors false` should not remove all your other jasmineNodeOpts), so we're going to leave this as-is for now. For the multiCapabilities issue, I feel that this is really hard to express via command line in a way that is intuitive and doesn't break other use cases, but I'm happy to take a look at a PR if anyone has a brilliant idea.